### PR TITLE
Fix simple-kmod e2e test to disable debug in not ready nodes

### PR DESCRIPTION
**Pull request by pacevedom**

### CODE REVIEW

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

File: test/e2e/basic/util.go [hunk 1]
```diff
@@ -286,3 +286,17 @@ func WaitForClusterOperatorCondition(cs *framework.ClientSet, interval, duration
 	}
 	return nil
 }
+
+// IsNodeReady helps determine if a given node is ready or not.
+func IsNodeReady(node corev1.Node) bool {
+	found := false
+	for _, condition := range node.Status.Conditions {
```
Comment (jmencak) [inline test/e2e/basic/util.go:293]: Is there a reason why we need to iterate over all the conditions even when we find `corev1.NodeReady`?  Would the following work?
```
// IsNodeReady helps determine if a given node is ready or not.
func IsNodeReady(node coreapi.Node) bool {
        for _, condition := range node.Status.Conditions {
                if condition.Type == coreapi.NodeReady {
                        return condition.Status == coreapi.ConditionTrue
                }
        }
        return false
}

```

Comment (dagrayvid) [reply test/e2e/basic/util.go:293]: Thanks Jiri, good catch. This looks to me to be totally equivalent but with less iterations.

Comment (pacevedom) [reply test/e2e/basic/util.go:293]: You are absolutely right Jiri, good catch!

**Pull request merged**